knowhere/knowhere/index/vector_index/impl/bruteforce/indexing/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
-set (CMAKE_CXX_STANDARD 11)
+set (CMAKE_CXX_STANDARD 17)
```

Comment:
guarantee CXX version is compatible with multiple platforms.

Reply:
This comment is removed. However could not find a liunx machine to test it out.